doc/reference/reference_lua/box_stat/net.rst

```diff
@@ -0,0 +1,43 @@
+==============================
```

Comment:
Wrong heading syntax. Please use single line of `=` under the text. https://www.tarantool.io/en/doc/latest/contributing/docs/markup/headings/

Reply:
Thanks, fixed